Tools/autotest/arducopter.py

```diff
@@ -1564,6 +1564,110 @@ def fly_nav_takeoff_delay_abstime(self):
         if ex is not None:
             raise ex
 
+    def fly_zigzag_mode(self):
+        '''test zigzag flight mode'''
```

Comment:
Triple double quote : """ instead of '''

Reply:
fixed, thanks.